docs/ice-dynamics.rst

```diff
@@ -1,7 +1,7 @@
-Ice dynamics
+Ice dynamic
```

Comment:
```suggestion
Ice dynamics
```

I think the plural form is correct https://en.wikipedia.org/wiki/Ice-sheet_dynamics

Reply:
Yes you're right. I though this opposite.